Assets/Scripts/DayCell.cs

```diff
@@ -1,29 +1,87 @@
-using System.Collections;
-using System.Collections.Generic;
+
+
 using UnityEngine;
 using TMPro;
 using UnityEngine.UI;
-using System;
+
 
 public class DayCell : Cell
 {
     [SerializeField] private TextMeshProUGUI cellText;
-
+    [SerializeField] private Image selectImage;
     [SerializeField] private Image defaultImage;
 
+    [SerializeField] private Image poop;
+    [SerializeField] private Image medicine;
 
+    private int clickCount = 0;
+    private DayCellData dData = new DayCellData();
+    public DayCellData DData => dData;
 
+    public override void Configure(CellData data)
+    {
+        dData = data as DayCellData;
+        SetTextValue(dData.text);
+        SetImageColor(dData.color);
+    }
 
+    private void Select()
+    {
 
-    public override void SetTextValue(string text)
+        if (clickCount == 0)
```

Comment:
Lets discuss this - was the initial plan not to have a click and hold? 

its not really possible but if clickCount =-1 or 2 this would never be triggered. This really should just be a bool. 

Should this not need double click to be in a certain timeframe

Reply:
Resolved

- I didn't want time limits for a second click